builtin/principals/local/Principals.go

```diff
@@ -42,7 +44,7 @@ func (p Principals) Get(ctx context.Context, payload []byte) (context.Context, [
 	}
 
 	if !p.UserMap.IsSet(local.User) {
-		return ctx, []string{}, fmt.Errorf("No principals found for %s", local.User)
+		return ctx, []string{}, princsPkg.NewNotFoundError("local", "No principals found for "+local.User)
```

Comment:
We don't need to add ldapPrinc.User to error, the logger already add it as a field.